src/osvr/Util/LogRegistry.cpp

```diff
@@ -87,22 +87,39 @@ namespace util {
             auto spd_logger = spdlog::get(logger_name);
```

Comment:
Rule #0 of library design: The library shall not cause the program to terminate.  No matter how bad things get, no matter that we think there's nothing the app can do about the situation, we return an error code rather than exiting or throwing an exception.